### PR TITLE
NEXT-339: Cleanup

### PR DESCRIPTION
**Pull request by SamRussellMM**
- Final cleanup of v2 Reporting Documentation
- Changing descriptions and parameter types to align with apiary doc

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -5233,9 +5237,11 @@ components:
         page_token:
           type: string
           example: abc123
+          description: Returned by Chronos service
```
Comment (liam-herbert-messagemedia) [inline spec/openapi.yaml]: Don't think we need this description on the public docs.

File: spec/openapi.yaml [hunk 2]
```diff
@@ -5395,38 +5416,52 @@ components:
...
+          description: Filter results for messages that include a metadata key containing this value. If this parameter is provided, the metadata_key parameter must also be provided.
         accounts:
-
           type: array
           items:
             type: string
           example:
             - Account1
             - Account2
+          description: Filter results by a specific account. By default results will be returned for the account associated with the authentication credentials and all sub accounts.
```
Comment (liam-herbert-messagemedia) [inline spec/openapi.yaml]: Should this be `sub-accounts` rather than `sub accounts`?

File: spec/openapi.yaml [hunk 3]
```diff
@@ -5395,38 +5416,52 @@ components:
...
           items:
             $ref: '#/components/schemas/MessageStatus'
+          example:
+            - DELIVERED
+            - ENROUTE
+          description: An array of message statuses.
         opt_out:
           type: string
           example: "true"
+          description: Filter the report to only include messages that triggered an opt out.
```
Comment (liam-herbert-messagemedia) [inline spec/openapi.yaml]: Should this be `opt-out` rather than `opt out`?

Comment (liam-herbert-messagemedia) [reply spec/openapi.yaml]: There are multiple occurrences of this elsewhere

**Review by liam-herbert-messagemedia — APPROVED** [review]

**Review by jamie-thomson-sinch — APPROVED** [review]

**Pull request merged**